Update to UFR

aaronbell opened the pull request:
I'm helping update repositories to Google's [United Font Repro](https://github.com/googlefonts/Unified-Font-Repository) format, as well as correcting STAT table issues in variable fonts. As an added benefit to this process, the font will now build automatically when PRs are accepted. 

Additionally:
- I made some modifications to the source metadata to enable it to build properly
- Underline thickness normalized due to known issue in certain scenarios. 

Once merged in, we'll submit an update to Google Fonts. Please let me know if you have any questions. Thanks!

aaronbell committed:
Updating to UFR

updated glyphs file and build process. bumped version string

aaronbell committed:
Update TRADEMARKS.md

aaronbell committed:
normalizing underline thickness

aaronbell committed:
Rebuild static fonts with autohinting

I made an error in my previous PR in disabling autohinting for the static instances. This PR corrects that as it doesn't affect the variable version. Fonts rebuilt.

m4rc1e commented:
ping @Omnibus-Type. Please merge @aaronbell's prs. His work greatly improves the fonts in applications.

aaronbell committed:
Various misc updates

Fixing the name back to "MuseoModerno", fixing some interpolation issues when running through fontmake.

Things generally better now.
